src/main/java/edu/ithaca/dragon/blackjack/HandReturn.java

```diff
@@ -1,42 +1,77 @@
 package edu.ithaca.dragon.blackjack;
 
 public class HandReturn {
```

Comment:
I think this class would be more appropriately named "RoundState" or something... It is not really a Hand anymore, it is really a class to encapsulate the current stat of a round to share with external entities, correct?